.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,24 +1,26 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: ''
-assignees: ''
-
+name: Feature Request
+about: Suggest a new feature or enhancement for this project
+title: ""
+labels: "Status: Needs Review"
+assignees: ""
```

Comment:
This info we need as it generates info on the actual issue selection screen as well as applies default labels/title to the issue.

So each time a user submits a "feature request" issue, it'll automatically have the "Status: Needs Review" label applied. Because the intent is for any issues that haven't been assigned yet to require reviewing from a maintainer to see whether the issue is valid/can be assigned, having this label auto-added just relieves some tedious upkeep that I and a couple others have been doing.

Reply:
ahh gotcha, TIL. Really neat 😄